jobspec/parse.go

```diff
@@ -401,6 +402,69 @@ func parseTasks(result *[]*structs.Task, list *ast.ObjectList) error {
 			}
 		}
 
+		if o := listVal.Filter("service"); len(o.Items) > 0 {
+			t.Services = make([]structs.Service, len(o.Items))
+			for idx, o := range o.Items {
+				var service structs.Service
```

Comment:
I would move all of this into a `parseService` function. See how we do it for others.